lessons/Algebra2/exploring-linear-models/langs/en-us/pages/graphing-models.adoc

```diff
@@ -0,0 +1,48 @@
+= Graphing Linear Models
+++++
+<style>
+
+.autonum { font-weight: bold; }
+.autonum:after { content: ')' !important; }
+
+td:first-child {
+	margin: 21px;
+	background-color: rgba(#fff, .5);
+	background-image:
+		linear-gradient(lightblue 2px, transparent 2px),
+		linear-gradient(90deg, lightblue 2px, transparent 2px),
+		linear-gradient(rgba(150, 150, 150, .3) 1px, transparent 1px),
+		linear-gradient(90deg, rgba(150, 150, 150, .3) 1px, transparent 1px);
+	background-size: 100px 100px, 100px 100px, 20px 20px, 20px 20px;
+	background-position: 56px 23px;
+}
+
+td:last-child { align-items: center; justify-items: center; }
+</style>
+++++
+
+Choose three linear models from @opt-printable-exercise{model-your-own}: your first guess, a closer guess, and then your *best* guess. For graph each model in the squares below, being sure to draw your axes. Then label the slope, y-intercept, and @math{R^2} value of each model!
+
```

Comment:
Oops - last sentence of para above needs to be fixed; doesn't make sense right now.